src/js/components/accordion/accordion.ts

```diff
@@ -19,10 +19,47 @@ export class AccordionBehavior extends Behavior {
         });
     }
 
+    toggleControl(target: HTMLElement, expanded?: boolean, attribute?: string): boolean {
```

Comment:
Is there ever a condition where you are passing in the third parameter attribute in? For what will you be using this third parameter for? So far while I was testing in storybook it seems to be always using aria-expanded?